src/index.js

```diff
@@ -1,23 +1,36 @@
 import './index.css';
+import TodoList, {
+  render,
+  inputTodo,
+  editDescription,
+} from './module/AddRemove.js';
+import './module/editableContent';
 
-const todoListContianer = document.querySelector('.todo__list');
+const TodoListObj = new TodoList();
+function Starter() {
+  const threeDots = document.querySelectorAll('li');
+  threeDots.forEach((dotValue, index) => {
+    const dot = dotValue.querySelector('.three__dots');
+    if (dot) {
+      const newDot = dot.cloneNode(true);
+      dotValue.replaceChild(newDot, dot);
+      newDot.addEventListener('click', () => {
+        editDescription(dotValue, index, TodoListObj);
+        Starter();
+      });
+    }
+  });
+}
 
-// const li = document.createElement('li');
```

Comment:
- [ ] Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.